apps/site/src/_pages/docs/3_using-blocks.mdx

```diff
@@ -27,6 +27,10 @@ The [Block Protocol for Wordpress](https://blockprotocol.org/wordpress) plugin a
 
 ### Other environments
 
-Support for additional embedding environments will arrive throughout 2023.
+Support for additional embedding environments will arrive throughout 2023. If there are others you'd like to see us exploring, please [reach out](https://blockprotocol.org/contact) to us.
 
-If you're looking to implement support for the Block Protocol in another application or framework that you'd like to see mentioned here, read more about [embedding blocks](https://blockprotocol.org/docs/embedding-blocks) and feel free to [reach out](https://blockprotocol.org/contact) to us.
+### Your own application
+
+If you're interested in supporting the Block Protocol within your application now, allowing your users to insert and use blocks from the [Þ Hub](https://blockprotocol.org/hub), please [contact us](https://blockprotocol.org/contact) with more information about your intended use, and we'll be in touch.
+
+You can also [register for updates](https://blockprotocol.org/signup) to find out as and when we release new docs and tools for embedding applications.
```

Comment:
```suggestion
You can also [register for updates](https://blockprotocol.org/signup) to get notified when we release new docs and tools for embedders.
```

Simplify language, and remove reference to 'embedding applications' as in theory it could be frameworks (like `tldraw`) which are used in other apps, as well as apps themselves.

Reply:
https://github.com/blockprotocol/blockprotocol/commit/a99d4f2457ac5c73af8b6b4d164455591eaf0b0a